x-pack/packages/ml/aiops_components/src/progress_controls/progress_controls.tsx

```diff
@@ -66,6 +66,7 @@ export function ProgressControls({
       <EuiFlexItem grow={false}>
         {!isRunning && (
           <EuiButton
+            data-test-subj={`aiopsRerunAnalysisButton${shouldRerunAnalysis ? ' shouldRerun' : ''}`}
```

Comment:
nit: Suggest to uppcase shouldRerun or add a dash as a prefix for readability (instead of `aiopsRerunAnalysisButtonshouldRerun`

Reply:
There is a space in the string so it should end up as `aiopsRerunAnalysisButton shouldRerun`.

Reply:
Aha I totally missed the extra space before the shouldRerun, in which case this reads fine. Just a small suggestion is to use a dash instead of space but this is just a super duper nit.

Reply:
AFAIK using the space would allow querying for the element with `~aiopsRerunAnalysisButton` even with `shouldRerunAnalysis` added, that's why I used the space, it wouldn't be possible with a dash.